build.gradle.kts

```diff
@@ -1,3 +1,5 @@
+import org.gradle.api.tasks.testing.logging.TestLogEvent
```

Comment:
Fix folder structure, it should look like this
![image](https://user-images.githubusercontent.com/55746901/222925703-8e687632-5a82-4d27-ab13-71e3968a51bd.png)

Reply:
Also, separate tests and test data, test data should be alongside src folder, since it can be used for demo